### ISSUE TITLE
Add bias option to tinygrad.nn.RMSNorm

### ISSUE THREAD

**Issue by EthanBnntt** [tinygrad/tinygrad#6349]

**Comment by EthanBnntt** [comment]
This is a feature that I would like to add.

**Comment by chenyuxyz** [comment]
torch adds RMSNorm in 2.4 https://pytorch.org/docs/2.4/generated/torch.nn.modules.normalization.RMSNorm.html, we might want to match that

**Comment by ryandorrington** [comment]
#6799

**Comment by wozeparrot** [comment]
Not correct, pr got reverted in #9052
